params/config.go

```diff
@@ -85,6 +85,7 @@ var (
 		PetersburgBlock:     big.NewInt(7_280_000),
 		IstanbulBlock:       big.NewInt(9_069_000),
 		MuirGlacierBlock:    big.NewInt(9_200_000),
+		RamanujanBlock:      big.NewInt(0),
```

Comment:
This should probably left as `nil` and not zero (which would mean the this fork is activated straight away)